lib/api/definitions/__tests__/CreatedMessage.test.ts

```diff
@@ -0,0 +1,349 @@
+      content: messageContent,
+      fiscal_code: "WRONG",
+      id: "12345",
+      sender_organization_id: "Sender Organization",
+      time_to_live: toTimeToLive(3600).get
+    };
+    expect(isCreatedMessage(messageOne)).toBe(false);
+  });
+
+  test("should returns true if CreatedMessage object does not have time_to_live property", () => {
+    const s = toMessageSubject("Lorem ipsum dolor sit amet");
+    const m = toMessageBodyMarkdown(
+      // String long 90 characters.
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt"
+    );
+
+    const messageContent: MessageContent = {
+      markdown: m.get,
+      subject: s.get
+    };
+
+    const messageOne = {
+      content: messageContent,
+      fiscal_code: toFiscalCode("AAABBB01C01A000A").get,
+      id: "12345",
+      sender_organization_id: "Sender Organization"
+    };
+    expect(isCreatedMessage(messageOne)).toBe(true);
+  });
+  test("should returns true if CreatedMessage object does have time_to_live property set to null", () => {
+    const s = toMessageSubject("Lorem ipsum dolor sit amet");
+    const m = toMessageBodyMarkdown(
+      // String long 90 characters.
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt"
+    );
+
+    const messageContent: MessageContent = {
+      markdown: m.get,
+      subject: s.get
+    };
+
+    /* tslint:disable */
+    const messageTwo = {
+      content: messageContent,
+      fiscal_code: toFiscalCode("AAABBB01C01A000A").get,
+      id: "12345",
+      sender_organization_id: "Sender Organization",
+      time_to_live: null
+    };
+    /* tslint:enable */
+    expect(isCreatedMessage(messageTwo)).toBe(true);
+  });
+  test("should returns false if CreatedMessage object does have time_to_live property mlformed", () => {
+    const s = toMessageSubject("Lorem ipsum dolor sit amet");
+    const m = toMessageBodyMarkdown(
+      // String long 90 characters.
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt"
+    );
+
+    const messageContent: MessageContent = {
+      markdown: m.get,
+      subject: s.get
+    };
+
+    const messageThree = {
+      content: messageContent,
+      fiscal_code: toFiscalCode("AAABBB01C01A000A").get,
+      id: "12345",
+      sender_organization_id: "Sender Organization",
+      time_to_live: "3600"
+    };
+    expect(isCreatedMessage(messageThree)).toBe(false);
+  });
+
+  test("should returns true if CreatedMessage object does not have content property", () => {
+    const messageOne = {
+      fiscal_code: toFiscalCode("AAABBB01C01A000A").get,
+      id: "12345",
+      sender_organization_id: "Sender Organization",
+      time_to_live: toTimeToLive(3600).get
+    };
+    expect(isCreatedMessage(messageOne)).toBe(true);
+  });
+  test("should returns true if CreatedMessage object does have content property set to null", () => {
+    /* tslint:disable */
```

Comment:
do not disable tslint altogether, instead just disable the specific rules you need to disable